decidim-meetings/app/views/decidim/meetings/meetings/show.html.erb

```diff
@@ -20,7 +20,16 @@
         <div class="extra__time">
           <%= meeting.start_time.strftime("%H:%M") %> - <%= meeting.end_time.strftime("%H:%M") %>
         </div>
-      </div>
+        <% if !meeting.closed? && meeting.registrations_enabled? %>
+          <% if meeting.has_registration_for? current_user %>
+            <%= action_authorized_button_to :leave, t('.going'), meeting_registration_path(meeting), method: :delete, class: "button expanded button--sc success" %>
+          <% else %>
+            <%= action_authorized_button_to :join, meeting.has_available_slots? ? t('.join') : t('.no_slots_available'), "", class: "button expanded button--sc", disabled: !meeting.has_available_slots?, data: { toggle: current_user.present? ? "meeting-registration-confirm" : "loginModal" } %>
+          <% end %>
+          <% if meeting.available_slots.positive? %>
+            <span><%= t(".remaining_slots", count: meeting.remaining_slots) %></span>
```

Comment:
Maybe we should display a text when there aren't remaining slots?

Reply:
It is already there: https://user-images.githubusercontent.com/106021/29366235-c32ecfa2-8299-11e7-8494-9bb86841a001.png